InteractionBase/src/commands/Command.cpp

```diff
@@ -25,11 +25,18 @@
 ***********************************************************************************************************************/
 
 #include "commands/Command.h"
+#include "precompiled.h"
 
 namespace Interaction {
 
+Command::Command(bool canBeUsedInMenu, QString name)
+    :canBeUsedInMenu_(canBeUsedInMenu), name_(name)
+{
+}
+
 Command::~Command()
 {
+    SAFE_DELETE(&name_);
```

Comment:
You must not delete value types. QString is a value type (it has not * in the definition). You don't need a destructor.